integration-cli/docker_cli_search_test.go

```diff
@@ -11,83 +12,50 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 	testRequires(c, Network, DaemonIsLinux)
 
 	out, exitCode := dockerCmd(c, "search", "busybox")
-	if exitCode != 0 {
-		c.Fatalf("failed to search on the central registry: %s", out)
-	}
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
 
-	if !strings.Contains(out, "Busybox base image.") {
-		c.Fatal("couldn't find any repository named (or containing) 'Busybox base image.'")
-	}
+	c.Assert(out, checker.Contains, "Busybox base image.", check.Commentf("couldn't find any repository named (or containing) 'Busybox base image.'"))
 }
 
 func (s *DockerSuite) TestSearchStarsOptionWithWrongParameter(c *check.C) {
 	out, exitCode, err := dockerCmdWithError("search", "--stars=a", "busybox")
-	if err == nil || exitCode == 0 {
-		c.Fatalf("Should not get right information: %s, %v", out, err)
-	}
-
-	if !strings.Contains(out, "invalid value") {
-		c.Fatal("couldn't find the invalid value warning")
-	}
+	c.Assert(err, check.NotNil, check.Commentf("Should not get right information: %s, %v", out, err))
+	c.Assert(exitCode, check.Not(check.Equals), 0, check.Commentf("Should not get right information: %s, %v", out, err))
+	c.Assert(out, checker.Contains, "invalid value", check.Commentf("couldn't find the invalid value warning"))
```

Comment:
I don't think there is a need for the Comment here since the error output will give the needed output.